### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by iower**
## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/swaponline/swap.react/wiki/CONTRIBUTING) guide
- [x] Styleguide check `npm run validate`
- [x] Good naming, keep simple
- [ ] Tested desktop/mobile
- [ ] Tested bright/dark
- [ ] Tested en/ru
- [ ] Affects money; I checked the functionality once again
- [x] I checked the PR once again

#3511 

- Clean github issue templates (only 3 variants: `bug`, `improvement`, `other`)
- Clean README

### CODE REVIEW

**Comment by noxonsu** [comment]
https://github.com/swaponline/MultiCurrencyWallet/pull/3509/checks?check_run_id=1333877205 чет не собралось

File: package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     "core:build:umd:win": "webpack --config webpack.config.babel.js",
     "test": "",
     "eslint": "eslint --fix --cache ./shared --ext js",
-    "validate": "lint-diff",
+    "linter": "lint-diff",
```
Comment (shendel) [inline package.json:29]: масло масленое

File: package.json [hunk 2]
```diff
@@ -71,7 +71,7 @@
   },
   "husky": {
     "hooks": {
-      "post-commit": "npm run common:test && npm run core:test"
+      "post-commit": "npm run common:test && npm run core:test && npm run linter"
```
Comment (shendel) [inline package.json:74]: без DEBUG.... - а оно вообще работает? 
image

**Review by shendel — CHANGES_REQUESTED** [review]
.

**Review by shendel — CHANGES_REQUESTED** [review]
.

**Comment by noxonsu** [comment]
@iower any updates?

File: package.json (hunk 1 above)
Comment (iower) [reply package.json]: Removed

File: package.json (hunk 2 above)
Comment (iower) [reply package.json]: Вidn't understand what the problem, but removed

**Review by noxonsu — APPROVED** [review]

**Pull request merged**